### ISSUE TITLE
The print designer does not work correctly with a text editor type field

### ISSUE THREAD

**Issue by 2old4thisShit** [frappe/print_designer#79]
When using a text editor field in the print designer, features such as changing the font do not work in the designer preview.

The change really works, but the preview does not show the change.

It seems to work correctly when you generate the pdf or print view. There the changes are displayed correctly.

Original:
![image](https://github.com/frappe/print_designer/assets/46288096/385f778b-755e-4e1e-8fd1-71c5a0c8bfed)

Original with alegreya font (no changes in view):
![image](https://github.com/frappe/print_designer/assets/46288096/3024e61b-71f9-46b0-ac4c-1042a71c6c12)

From print dialog (seems to be working):
![image](https://github.com/frappe/print_designer/assets/46288096/5a1576b5-7d33-4f51-afbc-66f62345dd83)

PDF View:
![image](https://github.com/frappe/print_designer/assets/46288096/4cc0e780-eaa3-4e8c-af59-3ef7a1336f65)

bench version:
erpnext 15.0.0
frappe 15.0.1
payments 0.0.1
print_designer 1.0.0
webshop 0.0.1

wkhtmltopdf --version
wkhtmltopdf 0.12.6.1 (with patched qt)

**Comment by maharshivpatel** [comment]
Can you check your network tab is there any error loading fonts. they are loaded from google fonts.

Is this PDF rendered on server or client i mean did you use download pdf button and downloaded the pdf or just pressed CMD/CTRL + P and used browsers printing.

**Comment by 2old4thisShit** [comment]
Network tab:
![image](https://github.com/frappe/print_designer/assets/46288096/28a8248b-8f3e-457d-a545-19268766493f)

I have used the buttons.

I only have the problem live, in the designer view.

Elsewhere, including here, it works perfectly:
![image](https://github.com/frappe/print_designer/assets/46288096/78edc1ac-9e16-4e97-9b36-9de805177cf3)

**Comment by maharshivpatel** [comment]
@2old4thisShit can you search fonts in network tab !

**Comment by 2old4thisShit** [comment]
this?
![image](https://github.com/frappe/print_designer/assets/46288096/a86ebcdb-8127-482f-b009-3d7c015aba78)

I made several font changes to test

**Comment by maharshivpatel** [comment]
@2old4thisShit  yeah, i just need to add !important for live preview if field is text editor. would you like to make a PR ? i can help you if you are interested else i will fix this in coming week

**Comment by 2old4thisShit** [comment]
Whatever you need :) tell me

**Comment by maharshivpatel** [comment]
by the way, this shouldn't have worked we can't just update the font because text we get is already formatted from the editor. it is a bug that changing the font works 😂 . we will try to add this if it always works

**Comment by 2old4thisShit** [comment]
Instead of a bug I have discovered a feature 😅🤣. Cool. The truth is that I was not sure if it was working correctly. So I tried my luck.

Actually with the text editor you can't change the font (I haven't tried it through code). It could be interesting to be able to do it from the print designer. Precisely because of the ease that the print designer adds by not having to use code.

Even if it's just the font... 🤷‍♂️, really dont know... xD
